website/docs/docs/dbt-versions/release-notes/09-April-2023/product-docs.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "April 2023 product docs updates"
+id: "april-product-docs"
+description: "April 2023: "
+sidebar_label: "Update: product docs updates"
```

Comment:
this is a personal pref - i feel it reads weird when the rn labele is update and then update is written again. another suggestion is 'product docs patches'  or 'product docs tweaks'. 

```suggestion
sidebar_label: "Update: product docs changes"
```